src/DIRAC/Core/Base/Script.py

```diff
@@ -55,6 +56,9 @@
 gIsAlreadyInitialized = False
 
 
+@deprecated(
+    "Do\nfrom DIRAC.Core.Utilities.DIRACScript import DIRACScript as Script\ninstead of\nfrom DIRAC.Core.Base import Script"
+)
 def parseCommandLine(script=False, ignoreErrors=False, initializeMonitor=False):
```

Comment:
How are we supposed to use DIRAC interactively in Python if we remove this? There is no equivalent to:

```python
from DIRAC.Core.Base.Script import parseCommandLine
parseCommandLine()
```

I'm very much in favour of having something more findable but `DIRACScript` isn't for use outside of DIRAC+extensions.

Reply:
I shared another method as we agreed at the last rally. Please, to use DIRAC interactively, use:
```python
from DIRAC import initialize
initialize()
```